pkg/apis/sources/v1/container_lifecycle_test.go

```diff
@@ -179,6 +195,7 @@ func TestContainerSourceStatusIsReady(t *testing.T) {
 			s.InitializeConditions()
 			s.PropagateSinkBindingStatus(&notReadySinkBinding.Status)
 			s.PropagateReceiveAdapterStatus(availableDeployment)
+			s.MarkOIDCIdentityCreatedFailed("unknown", "")
```

Comment:
```suggestion
			s.MarkOIDCIdentityCreatedSucceeded()
```

Reply:
Maybe we should put some cases where it gets failed, to make the tests more obedient.

Reply:
That would be great if you could add such a test